src/cider/nrepl/middleware/test/extensions.clj

```diff
@@ -0,0 +1,32 @@
+(ns cider.nrepl.middleware.test.extensions
```

Comment:
Btw, what's the idea behind having this in a separate namespace?

Reply:
Just so we can put functionality that extends `clojure.test`'s behavior in a separate place, since these customizations are by definition opinionated.

Reply:
Guess you can mention in the docstring this.

Reply:
👌